fullstacklabs/ohana-api
# Org Search/CS-200

[CS-200](https://fullstacklabs.atlassian.net/browse/CS-200)
**Needs the pagination branch on city-of-sac-client**
## Description

Create new org_search concern to enable searching through organizations for the categories associated with their services 
## Motivation and Context

Needed resources available feature and pagination functionality

## Issue Link
https://fullstacklabs.atlassian.net/browse/CS-200

## How Do I Test This?
Run db migration
Use postman to make a request to
`http://localhost:8080/api/organizations/search?category[]=Not+for+Profit&per_page=2&page=1`
Should return with two organizations.
